queenbee/schema/workflow.py

```diff
@@ -422,6 +472,56 @@ def hydrate_templates(workflow, wf_value=None):
     return wf_value
 
 
+def assert_workflow_values(values, references):
```

Comment:
Similar to the other case, since this is for internal use only we should either make it internal or make it  a `staticmethod` and part of the workflow class itself.